docs/deployment-examples/kubernetes-deployments/deploy-package/index.md

```diff
@@ -0,0 +1,621 @@
+---
+title: Deploy to a Kubernetes cluster
+description: Deploy to a Kubernetes cluster.
+position: 10
+---
+
+This featured was introduced as a prerelease in Octopus `2018.8`.
+
+:::warning
+Kubernetes steps in Octopus are of alpha level quality, and have been made available for testing and feedback purposes only. They must not be used for production deployments, or enabled on production Octopus instances. The information provided here is subject to change at any point, and existing Kubernetes steps will most likely need to be deleted and recreated with Octopus upgrades.
+:::
+
+Octopus supports the deployment of Kubernetes resources through the `Deploy Kubernetes containers` step. This step exposes a UI that builds up a [Kubernetes Deployment resource](http://g.octopushq.com/KubernetesDeploymentResource), a [Service resource](http://g.octopushq.com/KubernetesServiceResource), and an [Ingress resource](http://g.octopushq.com/KuberntesIngressResource). The combination of these resources represents an opinionated view about what makes up a typical Kubernetes deployment.
+
+## Deploy Kubernetes Containers Step
+
+To begin, add the `Deploy Kubernetes containers` step to a project.
+
+![Deploy Container Step](deploy-container-step.png)
+
+This step has three important sections that make up the combined objects that are deployed to Kubernetes.
+
+The first section is the `Deployment`. This section is used to build of the [Deployment resource](http://g.octopushq.com/KubernetesDeploymentResource).
+
+The second [feature](https://octopus.com/docs/deployment-process/configuration-features) is the `Service`. This feature is used to build a [Service resource](http://g.octopushq.com/KubernetesServiceResource).
+
+The third [feature](https://octopus.com/docs/deployment-process/configuration-features) is the `Ingress`. This feature is used to build a [Ingress resource](http://g.octopushq.com/KubernetesIngressResource).
+
+![Deploy Container Resources](deploy-container.svg)
+
+:::hint
+Kubernetes terminology overlaps with a number of general concepts in Octopus. For example, Kubernetes has the notion of a Deployment, which is distinct from the act of performing a deployment in Octopus.
+
+To distinguish between Kubernetes and Octopus terminology, we will reference to Kubernetes "resources" e.g. a Deployment resource or Pod resource.
```

Comment:
Is reference the correct word, or should it be 'refer'?

Reply:
Actually, I think it should just be 'we will reference ~to~ Kubernetes "resources"

Reply:
This has been fixed.